app/assets/javascripts/users.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
We recommend you create your models and controllers manually in this course. That will do two things for you:

1. You will learn a lot more about how models and controllers work.
2. You will not have a ton of extra files in your project like these.